### DISCUSSION TITLE
WETH wormhole does not transfer to Metamask

### DISCUSSION THREAD

**Discussion by jeta92** [wormhole-foundation/wormhole#387] [Problems / bugs] [answered]
WETH does not get back in my metamask wallet after sending it through wormhole. I sent it back from Sol to eth network through wormhole bridge, it does not show on my phantom and has not arrived to my metamask.
BV1NRU5FQvxKgY2e4jL4JKB1hiDrrTYYfhhiJpUJVhkJ

**Comment by leoluk** [comment, marked as answer]
Please provide a transaction hash

**Reply by leoluk** [reply]
I think this one may be it: https://explorer.solana.com/tx/5feQE9B3RHvK3DMSjCzESvv4MtoZRckqjfuWfCisFpP77z5SAj9UpK4mvUxQhKksudMitDhCYytBtERoY5jSUAhg

---

The transaction was successfully processed by the Wormhole network, but the client-side transaction submission may have failed. Try clearing your cache and reloading wormholebridge.com to see if the transaction can be completed there. Make sure both wallets are connected.

If wormholebridge.com doesn't work, you can manually submit the signed VAA bytes below to the submitVAA method on the Wormhole contract: https://etherscan.io/address/0xf92cd566ea4864356c5491c177a430c222d7e678#writeContract

---

Digest (for reference only): 1bc51e9e0f31a3709eaf26c5f9f30d57f81bf4f80d56ff2e320eecd569e1f14c

Signed VAA bytes to submit to the contract:

> 0x01000000010d0080d9743660060f5d287a8978dc0ed5ef9228b592a74e3adcb33229e6aea1fc53036711dda782433112e62fe4078e4d489062ee01f42f34c67e8428f86f012c790001b8d8d8bbf39e26361a466a11cb4b4052a5084a8677896435d5b4af1fd6744a6316086be1825458f98aee9efd5edffa13b68f8bc35b79807aec6de7aa9de2ccee01024a8a97313bc9ea0344cc85fbab159982f71abdd12e19f866894877e46c4d1b273a53298387b5740b9572e0c1fbbf522ce3aae7405ad9fd28053ad3dcda59e54d00053a667dc589c04ee5de5d8925a47146505c9ebef7d9a9f00ada2fedd852a0abec1c625c4bd9b1f33c807b651308e5c07da87deffd5367b0ac173261437fb5d01701067cbf95757839a2c97e283a691d1af95b56b1b5e3dcf29843644d7746ff31c41e076102e649c06b595264bd8150273dc79c3dd71c0c14e2ec953be8fc60a2cac90007aec65d9668c20d9f23e1743659c3a76f967510bd9020981db703aaac22b972f95fc8204a460b27bf6c9e01f7cfe8d2ac74c3409a4787bca784ca455a9d545fe801089be3f0f6f0a21566438e9825009ffccd9463487f68ebfa14a5ffda5d2ba9ca9b13f7f90f2365087bb0951d78d1c4d599ca8d9afa097e3ef18bdd65fdc626bdc00109a480633c89ae244a8bc15475ef19bd154ae379e7de993b9c689a98b8980d7cb835f82ccdccfbbc0ceacf1cf034203640693aebccd8aa35baa95c19ab135396af010a6effb3c9f72d8bafb163913a87d9aaa3adcc84582010651bb38e1d196c7115cf1b172e404f4d9a5b5f5d6d19760eef91b0c2d9ea29d58eb7f9eff4951afd9088010be98f199a8502a61891d36e3fd5c31468d94c6f8c85f798681dd3f47f835c2c535d7b9d27d98cd0781255c1a33d8b146595e41464e5754f35e78bb2550c67645c000d9175d1fc183519707fd565fab094daec87bd7893f588e880e650baefaa5aecd22f6fc9c4e12386cbb5416b1ce8c34f875fd7c7cad53c170fb7a15ca6b989368c0011341f615ef6cd131d80df56b812537890590f0a31af79c68d913d27b18f50af405a18847ef61ec8d4ceebd5d7bb077ac30eca2e2ee1b104445e61ca5b8684bdca0112fa4487f721ecb565a11645cea8c6328ec46febbb8b244e928cbcf265214f2bc83c82a6760cd38bc8ef6439678e914eda85ea98558afcfb55b6fc52bb0c0257a20061379f6b100001560f0102108b9bd1e7597785560cd848e1b9cdbf0ab8930711d5339377c4bd4aa7ec90c7000000000000000000000000df08d4cfa5af20d870d936499bc9bb65488587cd02000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc209000000000000000000000000000000000000000000000000000000001f972880
